src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -1361,9 +1360,41 @@ namespace EvaporativeFluidCoolers {
         int PltSizCondNum = state.dataPlnt->PlantLoop(this->plantLoc.loopNum).PlantSizNum;
         if (PltSizCondNum > 0) {
             this->DesignExitWaterTemp = state.dataSize->PlantSizData(PltSizCondNum).ExitTemp;
-            if (this->DesignEnteringWaterTemp == DataSizing::AutoSize) {
+        }
+
+        if (this->DesignEnteringWaterTempWasAutoSized && this->PerformanceInputMethod_Num == PIM::UserSpecifiedDesignCapacity) {
+            if (PltSizCondNum > 0) {
                 this->DesignEnteringWaterTemp =
                     state.dataSize->PlantSizData(PltSizCondNum).ExitTemp + state.dataSize->PlantSizData(PltSizCondNum).DeltaT;
+
+                if (this->DesignEnteringWaterTemp <= this->DesignEnteringAirWetBulbTemp) {
+                    ShowSevereError(
+                        state, format("Error when autosizing the Design Entering Water Temperature for Evaporative Fluid Cooler = {}.", this->Name));
+                    ShowContinueError(
+                        state,
+                        format("Design Entering Water Temperature ({:.2R} C) must be greater than design entering air wet-bulb temperature "
+                               "({:.2R} C).",
+                               this->DesignEnteringWaterTemp,
+                               this->DesignEnteringAirWetBulbTemp));
+                    ShowContinueError(
+                        state,
+                        "Check the Sizing:Plant object and the Design Entering Air Wet-bulb Temp input field for the Evaporative Fluid Cooler.");
+                    ShowFatalError(state, "Review and revise design input values as appropriate.");
+                }
+
+                if (state.dataPlnt->PlantFinalSizesOkayToReport) {
+                    BaseSizer::reportSizerOutput(
+                        state, this->EvapFluidCoolerType, this->Name, "Design Entering Water Temperature [C]", this->DesignEnteringWaterTemp);
+                }
+                if (state.dataPlnt->PlantFirstSizesOkayToReport) {
+                    BaseSizer::reportSizerOutput(
+                        state, this->EvapFluidCoolerType, this->Name, "Design Entering Water Temperature [C]", this->DesignEnteringWaterTemp);
+                }
```

Comment:
And we report this "Autosized" value to the EIO/SQL (IMHO all Autosize fields should report)